production/scrapers/california_statewide.R

```diff
@@ -1,16 +1,16 @@
 source("./R/generic_scraper.R")
 source("./R/utilities.R")
 
-california_statewide_pull <- function(x, wait = 15){
+california_statewide_pull <- function(x, wait = 25){
```

Comment:
I had to bump this up otherwise the numbers wouldn't load for me :( 

<img width="839" alt="Screen Shot 2021-06-21 at 9 46 15 AM" src="https://user-images.githubusercontent.com/47676353/122772584-876ce780-d275-11eb-9a3c-8162e0ea0cff.png">